File: example/fortran/ex.f
      double precision res(10)
      double precision x
      integer ireqs(10)
      call omnirpc_init
      x = 0.0
      do i = 1, 10
         call omnirpc_call_async(ireqs(i),'calc_sin*',x,res(i))
         x = x + 1.0
      end do
      call omnirpc_wait_all(10,ireqs)
      do i = 1, 10
         write(*,*) 'res(',i, ')=',res(i)
      end do 
      call omnirpc_finalize
      end

File: example/intro/ex.c
#include <OmniRpc.h>
#include <stdio.h>
#define MAX 200
int main(int argc,char *argv[])
{
   int i;
   double x, res[MAX];
	 OmniRpcRequest req[MAX];

   OmniRpcInit(&argc,&argv);

   for(i = 0; i < MAX; i++){
      req[i] = OmniRpcCallAsync("calc_sin",(double)i,&res[i]);
   }
   OmniRpcWaitAll(MAX,req);

   for(i = 0; i < MAX; i++)
      printf("sin(%d)=%g\n",i,res[i]);

   for(i = 0; i < MAX; i++){
      req[i] = OmniRpcCallAsync("calc_cos",(double)i,&res[i]);
   }
   OmniRpcWaitAll(MAX,req);

   for(i = 0; i < MAX; i++)
      printf("cos(%d)=%g\n",i,res[i]);

  OmniRpcFinalize();
}

  
